query-algebrizer/src/clauses/where_fn.rs

```diff
@@ -283,24 +308,45 @@ impl ConjoiningClauses {
             // The difference is that BindColl has only a single variable, and its values
             // are all in a single structure. That makes it substantially simpler!
             (Binding::BindColl(var), FnArg::Vector(children)) => {
+                if children.is_empty() {
+                    bail!(ErrorKind::InvalidGroundConstant);
+                }
+
                 // Turn a collection of arguments into a Vec of `TypedValue`s of the same type.
                 let known_types = self.known_type_set(&var);
                 // Check that every value has the same type.
                 let mut accumulated_types = ValueTypeSet::default();
+                let mut skip: Option<EmptyBecause> = None;
                 let values = children.into_iter()
```

Comment:
This comment might have gotten put in the wrong place; I'll try to make sure it's in the correct place as well.  Go Github!